Lib-Open5e/o5e_Token_rollAction.command

```diff
@@ -0,0 +1,43 @@
+[h: inputObj = arg (0)]
+[h: log.debug (getMacroName() + ": inputObj = " + inputObj)]
+[h: actionName = json.get (inputObj, "actionName")]
+[h: advDisadv = json.get (inputObj, "advDisadv")]
+[h: monsterActions = getProperty ("_o5e_MonsterActions")]
+[h, if (encode (monsterActions) == ""), code: {
+	[broadcast ("No Monser Actions object", "self")]
+	[return (0)]
+}; {""}]
+[h: actionObj = json.get (monsterActions, actionName)]
+[h, if (encode (actionObj) == ""), code: {
+	[broadcast ("Action " + actionName + " not found")]
+}; {""}]
+[h: actionObjType = json.type (actionObj)]
+[h: log.debug ("actionObjType = " + actionObjType)]
+[h: output = "Something <i>HAPPENS</i>"]
+[h, if (actionObjType == "UNKNOWN"), code: {
```

Comment:
Mysterious "Something Happens" is sufficient instead of building fake saveable. Not even sure this applies anymore; remove.